### DISCUSSION TITLE
Floating Window Colors

### DISCUSSION THREAD

**Discussion by joe23rep** [nvim-tree/nvim-tree.lua#1542] [Q&A]
Im creating a new colorscheme right now and would like to add specific colors for nvimtree. 

Set up everything i wanted so far but i cant find out what highlight group you use for the floating window border. 

Would be nice if you could add that to the documentation

**Comment by alex-courtis** [comment]
nvim-tree doesn't actually do any styling of the floating window; it's just regular vim styling.

Take a look at [base16-vim](https://github.com/chriskempson/base16-vim) for inspiration.
